my_board.h

```diff
@@ -13,19 +13,20 @@ class board{
 		board(board * oldBoard);									//copy constructor
 		~board();													//destructor (delete all pieces etc.)
 
-		AbstractPiece * GetPiece(int raw, int col){return thisboard[raw][col];};	//returns piece from raw and column
-		bool Occupied(int n_raw, int n_col){return (thisboard[n_raw][n_col] -> GetType() != 0);};//is a position occupied?
+		AbstractPiece * GetPiece(int raw, int col){return thisboard[raw][col];};
+		bool Occupied(int n_raw, int n_col){return (thisboard[n_raw][n_col] -> GetType() != 0);};
 
-		void Print();												//prints the board
+		void Print();												
 		string Unicode(type_piece type, color_piece color); 		//converts type+color in unicode char
 
 		int GetCurrentMoveInt(){return currentmove;};				//returns 0 || 1 for white & black
-		void SetCurrentMoveInt(int a){currentmove = a;}	;		//set currentmove
-		void InvertCurrentMoveInt(){if (currentmove==0)	currentmove=1; else	currentmove=0;}; //Invert 0.1 currentmove
+		void SetCurrentMoveInt(int a){currentmove = a;};
+		void InvertCurrentMoveInt(){if (currentmove==0)	currentmove=1; else	currentmove=0;};
```

Comment:
you can use ternary operator for if inline
https://www.programiz.com/cpp-programming/ternary-operator
```
currentmove = (currentmove==0) ? 1 : 0;
```